spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/flyway/FlywayAutoConfiguration.java

```diff
@@ -110,6 +111,13 @@ else if (this.flywayDataSource != null) {
 			else {
 				flyway.setDataSource(this.dataSource);
 			}
+			flyway.setSqlMigrationPrefix(properties.getSqlMigrationPrefix());
+			flyway.setSqlMigrationSuffix(properties.getSqlMigrationSuffix());
+			flyway.setValidateOnMigrate(properties.isValidateOnMigrate());
+			flyway.setBaselineOnMigrate(properties.isBaselineOnMigrate());
+			flyway.setIgnoreFailedFutureMigration(properties.isIgnoreFailedFutureMigration());
+			flyway.setTable(properties.getTable());
+			flyway.setBaselineDescription(properties.getBaselineDescription());
 			return flyway;
```

Comment:
Like I said in the issue `Flyway` is exposed so besides improving the documentation, this should already work.